api/src/test/java/org/openmrs/api/db/hibernate/search/LuceneAnalyzerFactoryTest.java

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
This is the relevant file in this PR; the others are from https://github.com/openmrs/openmrs-core/pull/3040